packages/gestalt/src/TapArea.js

```diff
@@ -189,6 +191,7 @@ const TapAreaWithForwardRef: React$AbstractComponent<
         onMouseLeave={handleLinkOnMouseLeave}
         ref={innerRef}
         rel={rel}
+        tabIndex={tabIndex}
```

Comment:
@AlbertCarreras we should also add `tabIndex` to the `<div />` element below.

If the TapArea is disabled, it should always be `null`
If the TapArea is not disabled, it should default to `0` (TAB-able) but if the `tabIndex` prop set to `-1`, it should be `-1`

Reply:
done